Sources/Rendering/Core/Camera/index.js

```diff
@@ -343,9 +343,67 @@ function vtkCamera(publicAPI, model) {
   publicAPI.setObliqueAngles = (alpha, beta) => {};
   publicAPI.getOrientation = () => {};
   publicAPI.getOrientationWXYZ = () => {};
-  publicAPI.getFrustumPlanes = (aspect) => {
-    // Return array of 24 params (4 params for each of 6 plane equations)
+
+  publicAPI.getFrustumPlanes = (aspect = 1.0) => {
+    // Allocate normals as a flat array (6 planes * 4 components)
+    const normals = new Float64Array([
+      // Left
+      1, 0, 0, 1,
+      // Right
+      -1, 0, 0, 1,
+      // Bottom
+      0, 1, 0, 1,
+      // Top
+      0, -1, 0, 1,
+      // Near
+      0, 0, 1, 1,
+      // Far
+      0, 0, -1, 1,
+    ]);
+
+    const planes = new Float64Array(24);
+
+    // Get the composite projection matrix
+    const matrix = publicAPI.getCompositeProjectionMatrix(aspect, -1, 1);
+
+    // Transpose the matrix for use with normals
+    const transposed = new Float64Array(16);
```

Comment:
Please use glMatrix or vtkMath to transpose a matrix instead of writing it yourself.